packages/venia-ui/lib/util/makeUrl.js

```diff
@@ -76,7 +83,14 @@ const makeOptimizedUrl = (path, { type, ...opts } = {}) => {
     // Append image optimization parameters
     const params = new URLSearchParams(baseURL.search);
     params.set('auto', 'webp'); // Use the webp format if available
-    params.set('format', 'pjpg'); // Use progressive JPGs at least
+
+    const imageFileType = getFileType(baseURL);
+    if (imageFileType === 'png') {
+        params.set('format', 'png'); // use png if webp is not available
```

Comment:
Piggybacking on the [comment](https://github.com/magento/pwa-studio/issues/2340#issuecomment-625337800) @Jordaneisenburger made on the issue, what should we do to make this extensible?

Should we do something like this?

```es6
if (isValidImageFileType(imageFileType)) {
    params.set('format', imageFileType);
} else {
    params.set('format', 'pjpg');
}
```